### PR TITLE
watch later page

### PR DESCRIPTION
**Pull request by nerdyspook**
**Added Watch Later Page**
- Users can save videos on the watch-later page in order to view them later.
- Users can remove videos from watch-later by clicking on the watch-later button again.
- Users can remove videos from watch-later by going to the watch later page and clicking on the remove button.

**Minor changes**
- when a user clicks on a suggested video on the watch page, that video will get added to the history page as well.
- remove button will only appear on the history and watch-later page.

### CODE REVIEW

File: src/components/VideoCard/VideoCard.jsx [hunk 1]
```diff
@@ -1,27 +1,54 @@
-import React from "react";
+import React, { useState } from "react";
 import { BsThreeDotsVertical } from "react-icons/bs";
 import { IoRemoveCircle } from "react-icons/io5";
 import { useLocation, useNavigate } from "react-router-dom";
 import { useVideo } from "../../context/VideoContext";
 import { addToHistory } from "../../utilities/History/add-history";
 import { removeFromHistory } from "../../utilities/History/remove-history";
+import { addToWatchLater } from "../../utilities/WatchLater/add-watch-later";
+import { removeFromWatchLater } from "../../utilities/WatchLater/remove-watch-later";
```
Comment (KanekarSnehal) [inline src/components/VideoCard/VideoCard.jsx:9]: You can make an index.js file in utilities and import all files from there

**Review by KanekarSnehal — APPROVED** [review]
Code is well structured
Liked the use of pathname

File: src/components/VideoCard/VideoCard.jsx (hunk 1 above)
Comment (nerdyspook) [reply src/components/VideoCard/VideoCard.jsx:9]: Sure, will do that while refactoring the code.

**Review by shivanigangadharan — APPROVED** [review]
Code looks great, I like that you've added good comments for each function. Please also add a deploy preview or gif for better understanding of UI.

**Pull request merged**